wurst/systems/buildings/SpiritWard.wurst

```diff
@@ -43,10 +45,11 @@ init
                 ward.setLife(0)
                 setWardStoppedReviving(ward)
 
-    registerPlayerEvent(EVENT_PLAYER_LEAVE) ->
-        let leaver = GetTriggerPlayer()
-        let wards = GetUnitsOfPlayerAndTypeId(leaver, UNIT_SPIRIT_WARD)
-        let tribe = GetPlayerTribe(leaver)
+    registerGameStartEvent() ->

```

Comment:
Add `TODO` comment here saying we need to accommodate the revival of trolls of players who leave.